README.md

```diff
@@ -1931,6 +1931,181 @@ parent       <uuid>
  ```
 
 
+### **Commands V2**
+Commands V2.0 command is used to list the command requests, statuses, fire different actions on devices or groups like ping, install application etc. It provides advanced device commands capabilities like queuing, support for offline devices, dynamic device set for commands and command history.
+
+```sh
+$ espercli commandsV2 [SUB-COMMANDS]
+```
+#### Sub commands
+#### 1. list
+List and filter command requests.
+```sh
+$ espercli commandsV2 list [OPTIONS]
+```
+##### Options
+| Name, shorthand     | Default| Description|
+| ------------------- |:------:|:----------|
+| --commandtype, -ct  |        | Filter by type of command request |
```

Comment:
Use snake case

Reply:
Okay

Reply:
Also in description mention default values available for each parameter. And start with a simple example with minimum required options and then give an example by changing default variables.

Reply:
Okay. Will do.